selection/randomized/convenience.py

```diff
@@ -87,6 +87,7 @@ def __init__(self,
 
         self.covariance_estimator = covariance_estimator
 
+        self.randomizer_scale = randomizer_scale
```

Comment:
Do we need to store this or is this only for the test?

We can pass the scale to the convenience constructor for the test if we want.

Reply:
This is only for the test since we don't keep the randomizer_scale anywhere so I wasn't sure how to recover it.